activities/tool-management/index.md

```diff
@@ -19,11 +19,17 @@ This guidance covers:
 
 Our current tools are:
 
+* [1password](1password.md) (password management)
 * [Docusign](docusign.md) (contract management)
+* [GitHub](github.md) (version control and codebase management)
 * [Google Workspace](google-workspace.md) (email, calendaring and internal document management)
 * [Jitsi Meet](jitsi-meet.md) (videoconferencing)
+* [Mailman](mailing-lists.md) (mailing lists with Hyperkitty archiver)
 * [Mattermost](mattermost.md) (chat)
+* [Miro](miro.md) (online collaborative whiteboard)
+ [Nextcloud](nextcloud.md) (file hosting services)
 * [Odoo Community](odoo.md) (CRM and project management)
 * [Podbean](podbean.md) (publishing our podcasts)
 * [Streamyard](streamyard.md) (livestreaming our podcast)
 * [Tentoo](tentoo.md) (human resources management, including processing salaries and expenses)
+* [Voys](voys.md) (VoIP)
```

Comment:
Can I suggest saying 'Telephony' since it is probably understood wider than VoIP.

```suggestion
* [Voys](voys.md) (Telephony)
```